### PR TITLE
More conventions and markdown fixes

### PR DESCRIPTION
**Pull request by renatoliveira**
I've specified more conventions, like indentation and the use of white space. Also fixed markdown issues (pointed by VS Code).

### CODE REVIEW

**Review by gitmatheus — APPROVED** [review]
Very good additions.

**Review by gitmatheus — APPROVED** [review]

**Review by gitmatheus — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,27 +58,43 @@
...
+
 ```java
 public static final String PAYPAL_LOGIN_URL = 'https://login.paypal.com/';
 ```
 
 ## 5. SOQL and SOSL queries
 
-* SOQL and SOSL commands should be in uppercase:
+* SOQL and SOSL keywords should be uppercase.
+* Fields to retrieve should be on separate lines, with the separating comma on the same line as the field.
```
Comment (gitmatheus) [inline README.md]: This is only applicable when there are several fields in the Query. When the query is retrieving just a few fields, than its preferable to have it in the same line.

**Review by gitmatheus — CHANGES_REQUESTED** [review]
This is only applicable when there are several fields in the Query. When the query is retrieving just a few fields, than its preferable to have it in the same line. 

That's the only thing I disagree. Once it's fixed, I'll approve the Pull Request.

File: README.md (hunk 1 above)
Comment (renatoliveira) [reply README.md]: I've modified the text a bit to reflect small and bigger SOQL statements.

**Review by gitmatheus — APPROVED** [review]

**Pull request merged**